modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue

```diff
@@ -54,7 +54,22 @@
         />
       </div>
       <div class="apos-admin-bar__row">
-        <div class="apos-admin-bar__context-spacer" />
+        <div class="apos-admin-bar__context-controls">
+          <AposButton
+            :disabled="patches.length === 0"
+            type="default" label="Undo"
+            class="apos-admin-bar__context-button"
+            icon="undo-icon" :icon-only="true"
+            @click="undo"
+          />
+          <AposButton
+            :disabled="undone.length === 0"
+            type="default" label="Redo"
+            class="apos-admin-bar__context-button"
+            icon="redo-icon" :icon-only="true"
+            @click="redo"
+          />
```

Comment:
THese need tooltips

Reply:
Also, use the undo/redo variants

![image](https://user-images.githubusercontent.com/1889830/100018013-c2736f00-2da9-11eb-9fa4-cb9aab0e0674.png)

Reply:
Also also, these buttons should use the `outline` and `no-motion` modifiers